src/core/AudioEngine.cpp

```diff
@@ -1157,10 +1162,10 @@ MidiClient * AudioEngine::tryMidiClients()
 #endif
 
 #ifdef LMMS_BUILD_WIN32
-	if( client_name == MidiWinMM::name() || client_name == "" )
+	if (client_name == MidiWinMM::name() || client_name == "")
 	{
 		MidiWinMM * mwmm = new MidiWinMM;
-//		if( moss->isRunning() )
+//		if ( moss->isRunning() )
```

Comment:
Just a comment, but since you've already started formatting it:
```suggestion
//		if (moss->isRunning())
```